source/_integrations/spotify.markdown

```diff
@@ -113,6 +115,32 @@ script:
 
 The `media_content_id` value can be obtained from the Spotify desktop app by clicking on the more options ("...") next to the album art picture, selecting "Share" and then "Copy Spotify URI" or "Copy Playlist Link" (also available in the Spotify phone and web app).
 
+The `media_content_id` can be a URL (e.g. `https://open.spotify.com/playlist/37i9dQZF1DWVFeEut75IAL`) or the Spotify URI string (e.g. `spotify:playlist:37i9dQZF1DWVFeEut75IAL`). (play_media)[https://www.home-assistant.io/integrations/media_player/#service-media_playerplay_media] requires the correct corresponding `media_content_type`.
+
+## Selecting output source
+
+To play media Spotify needs a device selected for audio output known as the `source`. Currently, this integration cannot initiate playback to a device not already available from the Spotify API. You need to first use Spotify on your smart speaker or phone to populate the source list.
+
+```yaml
+# Example code to select an AV receiver as the output device
+service: media_player.select_source
+entity_id: media_player.spotify
+data:
+  source: "Denon AVR-X2000"
+```
+
+The device name must be known to Spotify and therefore is often different from names given to entities in Home Assistant.
```

Comment:
What does this mean?

Reply:
It means if Spotify has my phone named "Kevs S10 but Home Assistant calls it "SM-G973F", the latter is irrelevant.

I think I over emphasis that this is the name in the Spotify API, _not_ Home Assistant. Initially it was not obvious to me that 
- HA tells Spotify API to play on a Device
- NOT HA steams to a Device from Spotify API 

But all of that is covered below so I will simply this 😄